### PR TITLE
updated gradle dependencies

### LINKED ISSUE #2366: Update library dependencies in the application.
**Steps to reproduce it**

There are many library dependencies for the application that needs to be updated to newer and latest versions. This gives us the latest and more stable code from the libraries and thus makes the application more stable. 

**Would you like to work on the issue?**
Yes

**Target Milestone to solve the issue**
It can be solved in a small fix.

### PR DESCRIPTION
**Pull request by sriramr98**
Fixes #2366
Changes: updated versions of updated dependencies.

### CODE REVIEW

File: android/app/build.gradle
Comment (iamareebjamal) [inline android/app/build.gradle]: Can you move this below rxjava. RxJava deps should be together

File: android/app/build.gradle
Comment (iamareebjamal) [inline android/app/build.gradle]: Add space after `//`

**Review by iamareebjamal — APPROVED** [review]

**Comment by sriramr98** [comment]
@iamareebjamal if all is fine, merge it with development and close the PR

**Comment by iamareebjamal** [comment]
It takes time to run the build after approving it, we can only merge after build has passed

**Pull request merged**
